Core/src/org/sleuthkit/autopsy/persona/PersonaDetailsTopComponent.java

```diff
@@ -18,35 +18,28 @@
  */
 package org.sleuthkit.autopsy.persona;
 
+import org.openide.windows.TopComponent;
 import org.openide.util.NbBundle.Messages;
 import org.openide.windows.RetainLocation;
-import org.openide.windows.TopComponent;
 import org.openide.windows.WindowManager;
 
 /**
- * Top component for the Personas tool
- *
+ * Top component for persona details
  */
 @TopComponent.Description(preferredID = "PersonasTopComponent", persistenceType = TopComponent.PERSISTENCE_NEVER)
-@TopComponent.Registration(mode = "personas", openAtStartup = false)
-@RetainLocation("personas")
+@TopComponent.Registration(mode = "personadetails", openAtStartup = false)
+@RetainLocation("personadetails")
 @SuppressWarnings("PMD.SingularField")
-public final class PersonasTopComponent extends TopComponent {
+public final class PersonaDetailsTopComponent extends TopComponent {
 
     @Messages({
-        "PTopComponent_Name=Personas"
+        "PTopComponent_Name=Persona Details"
```

Comment:
I'm getting a build error.  PTopComponent_Name is key is being duplicated in two files.

warning: Supported source version 'RELEASE_7' from annotation processor 'org.netbeans.modules.openide.util.NbBundleProcessor' less than -source '1.8'
C:\Basis\Code\autopsy\Core\src\org\sleuthkit\autopsy\persona\PersonaSearchTopComponent.java:49: error: Duplicate key: PTopComponent_Name
    public PersonaSearchTopComponent() {
C:\Basis\Code\autopsy\Core\src\org\sleuthkit\autopsy\persona\PersonaDetailsTopComponent.java:38: error: Duplicate key: PTopComponent_Name
    public PersonaDetailsTopComponent() {